articles/machine-learning/how-to-manage-quotas.md

```diff
@@ -112,10 +113,15 @@ Azure Machine Learning managed online endpoints have the following limits.
 | Number of instances per deployment | 20 |
 | Max payload size at endpoint level |1.5 MB |
 | Max request time out at endpoint level  | 60 seconds |
-| Total QPS at endpoint level for all deployments  | 100 |
-
-<sup>1</sup> Single dashes like, `my-endpoint-name`, are accepted in endpoint and deployment names
+| Total requests per second at endpoint level for all deployments  | 100 <sup>2</sup> |
+| Total connections per second at endpoint level for all deployments  | 25 <sup>2</sup> |
+| Total connections active at endpoint level for all deployments  | 20 <sup>2</sup> |
+| Total bandwidth at endpoint level for all deployments  | 500 KBPS <sup>2</sup> |
 
+<sup>1</sup> Single dashes like, `my-endpoint-name`, are accepted in endpoint and deployment names.
+ 
+<sup>2</sup> If making request for a limit increase, make sure to also calculate and include all the other limit increases that you might need. E.g. if asking to increase the RPS limit to 200, you might also want to ask for increasing the active connections and bandwidth limit as per your requirement.
```

Comment:
Can we eleborate on this point a little bit more or maybe even add separate section (e.g. how different limits are connected)?

Reply:
Different limits are not really connected e.g. increasing qps limit may or may not need increased bandwidth or connection limits. The user is best to make this determination based on their request and client characteristic. That's why I chose to provide a hint to serve as a reminder for them to think about all the different aspects before making the request ticket. Thoughts?